benchmarks/tensorexpr/__main__.py

```diff
@@ -92,7 +92,15 @@ def main():
     if args.cuda_fuser == "te":
         import torch
 
+        torch._C._jit_set_profiling_executor(True)
         torch._C._jit_set_texpr_fuser_enabled(True)
+        torch._C._jit_set_profiling_mode(True)
+    elif args.cuda_fuser == "old":
+        import torch
+        torch._C._jit_set_profiling_executor(False)
+        torch._C._jit_set_texpr_fuser_enabled(False)
+        torch._C._jit_set_profiling_mode(False)
```

Comment:
Nit: technically `set_profiling_mode` is unneeded after we call `_jit_set_profiling_executor(False)`. Also, since the fuser was specifically requested to be "old" it might make sense to turn it on explicitly rather than relying on a default value (` torch._C._jit_override_can_fuse_on_gpu(True)`)